src/components/shell.js

```diff
@@ -94,13 +102,19 @@ export default function Shell({ children, defaultStatus = 'default' }) {
       }
     }
 
+    // ensure we render height as side-effect of page load
+    renderShellHeight();
+
+    const debouncedWindowResize = debounce(renderShellHeight, 500);
```

Comment:
need to call `debouncedWindowResize.cancel()` when we cleanup

Reply:
good catch! added.